<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Waffle ADF Disk Copier</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/ArduinoFloppyReaderWin.ico</normaloff>res/ArduinoFloppyReaderWin.ico</iconset>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>572</width>
     <height>414</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,1,0,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QFrame" name="topLayout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>561</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
         <item>
          <layout class="QVBoxLayout" name="WafflLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Amiga Floppy Disk Reader, Created by Gianluca Renzi,
(C) 2019 RetroBit Lab</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Waffle is connected on COM:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serialPort">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>type USB/COM Port number here...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Arial Black</family>
               <pointsize>42</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>res/Waffle-logo-small.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lRobSmithLogo">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>res/logo_retrobitlab-small.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Copy disk to ADF File (read)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="midLayout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>541</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLineEdit" name="fileName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string extracomment="Write File Name"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>type here file name to save as ADF File...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="readButton">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Copy Disk</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="bottomLayout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="horizontalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>-1</y>
              <width>481</width>
              <height>41</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,1,0,1,0">
             <item>
              <widget class="QLabel" name="currTrack">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Current Track:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="track">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="currSide">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Current Side:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="side">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Upper</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="badSect">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Bad Sectors:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="bad">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QFrame" name="frame_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>549</width>
              <height>36</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QProgressBar" name="progressBar">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>531</width>
               <height>23</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
            <zorder>progressBar</zorder>
            <zorder>shader</zorder>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="finishGood">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>171</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/readComplete.bmp</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="finishBad">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>281</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>281</width>
      <height>171</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/readPartial.bmp</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ClickLabel" name="shader">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QLabel</extends>
   <header location="global">clicklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
